content/docs/reference/glossary.md

```diff
@@ -6,54 +6,208 @@ redirectFrom:
   - /docs/cloud/concepts/
 ---
 
+## A
+
+<a id="access-token">Access token</a>
```

Comment:
This doesn't seem right to me 😄  It violates the HTML semantics, as these are not quite links but rather just anchors. Please, use `###`, or 3rd-level heading, which is going to be added to ToC automatically. It will look a bit better 😉 

```suggestion
### Access token
```